Documentation/getting-started.md

```diff
@@ -20,18 +20,22 @@ Getting started
 git clone https://github.com/stratisproject/StratisBitcoinFullNode.git  
```

Comment:
since editing the file anyway, might want to fix the misspelling of 'repository' on line 18?